timelock-agent/src/main/java/com/palantir/timelock/paxos/TimelockPaxosLearnerRpcClient.java

```diff
@@ -54,20 +54,20 @@ void learn(
             @PathParam("seq") long seq,
             PaxosValue val);
 
-    @Nullable
+    @Nonnull
     @GET
     @Path("learned-value/{seq}")
     @Produces(MediaType.APPLICATION_JSON)
-    PaxosValue getLearnedValue(
+    Optional<PaxosValue> getLearnedValue(
             @PathParam("useCase") PaxosUseCase paxosUseCase,
             @PathParam("client") String client,
             @PathParam("seq") long seq);
 
-    @Nullable
+    @Nonnull
```

Comment:
Don't think we need the `@Nonnull` annotations, since that should be implied with code wide `Nonnull` by default and also the fact that it's a huge code smell to pass around a `null` `Optional`

Reply:
This was done for consistency with the other methods in the client. I guess now all of them are nonnull it's probably fine to remove them.